core/src/main/java/org/eclipse/krazo/cdi/AroundController.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2017, 2018 Ivar Grimstad
```

Comment:
I think this should be ...2017, 2019... according to mail discussions on the Jakarta EE Community mailing list...

Reply:
...or actually only ...2019..., I guess since it being a new file... the changed files should be ...2017, 2019...